common/src/abi/mod.rs

```diff
@@ -75,4 +101,15 @@ mod test {
             }
         }
     }
+
+    #[test]
+    fn test_pascal_case() {
+        assert_eq!(pascal_case_split("TestOne"), vec!["Test", "One"]);
+        assert_eq!(
+            pascal_case_split("TestFailWithMultipleWords"),
```

Comment:
Just wondering why we use the word _fail_ in this string? - given the test does not fail, right?

Reply:
`TestFail` is one of the `Method`s, see https://github.com/polyphene/kythera/issues/24. That's why I used it there.

Reply:
Makes sense 👍. Thank you.